web/pdf_page_view.js

```diff
@@ -223,6 +233,19 @@ class PDFPageView {
     }
   }
 
+  addLayer(div, name) {
```

Comment:
With the other comments addressed, regarding `onAppend`, this can become a *private* method instead.